src/javaharness/java/arcs/api/Thing.java

```diff
@@ -18,4 +20,12 @@ public boolean equals(Object other) {
     public int hashCode() {
         return get().hashCode();
     }
+
+    public NativeParticle getParticle() {
+        return this.thing instanceof NativeParticle ? Optional.of((NativeParticle)this.thing).orElse(null) : null;
```

Comment:
space after paren
(NativeParticle) this

Reply:
done